### PR TITLE
New images

### PR DESCRIPTION
**Pull request by ccorley**
Added build instructions for JetStream and Kong

### CODE REVIEW

File: nats-server/README.md [hunk 1]
```diff
@@ -1,13 +1,115 @@
 # Linux for Health NATS Server Image
 
-The Linux for Health NATS Server Image provides event notifications via message streaming.
+The Linux for Health NATS Server Image provides event notifications via NATS JetStream message streaming.
 
-## Build Command
+## Overview
+NATS JetStream is based on two dependent images, jq for JSON parsing and nats-box for CLI capabilities.  Pre-built images for JetStream, jq and nats-box are only available outside of LinuxForHealth as amd64 images.  The instructions below explain how to build all 3 of these images on all 3 LinuxForHealth-supported chip platforms (amd64, s390x & arm64).
 
+Because `docker buildx` has problems building these images on all 3 platforms, particularly s390x, we build each image natively, then create a multi-arch image with `docker manifest`.  So to rebuild these images, you'll meed amd64, s390x & arm64 build environments.
```
Comment (dixonwhitmire) [inline nats-server/README.md:8]: Should we update our other build processes/instructions to use the docker manifest based process?

Comment (ccorley) [reply nats-server/README.md:8]: I don't think so.  Where it works, we should use docker buildx.  It just didn't work in any of these cases.  But buildx is much easier when it does work.

**Review by dixonwhitmire — APPROVED** [review]
Great work on updating NATS Jetstream and supporting multi-arch builds! In a future iteration it would be worthwhile to consider installing jq, if available through the apk package manager in alpine. That would remove the dependency on the additional builder image and could be something we contribute back to NATS proper if the jq version is compatible, etc.

**Comment by ccorley** [comment]
Good point on the apk for jq.  Not sure if there is an apk for all platforms, but we can look into that when we restructure the images.

**Pull request merged**